src/components/Home.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Each from './Each';
+
+const Home = function myProject5({ nasdaq }) {
+  const companies = nasdaq;
+  companies.sort((a, b) => b.changesPercentage - a.changesPercentage);
+  return (
+    <>
+      <div className="full-width">
+        <h5 className="list-heading">Top Performers</h5>
+      </div>
+      <div className="container">
+        {companies.map((item, index) => (
+          // eslint-disable-next-line max-len
```

Comment:
- [ ] Kindly remove this eslint disable line its best to fix the issue in the code.